mlflow/utils/download_cloud_file_chunk.py

```diff
@@ -0,0 +1,56 @@
+"""
+This script should be executed in a fresh python interpreter process using `subprocess`.
+"""
+import argparse
+import importlib.util
+import json
+import os
+import requests
+import sys
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--range-start", required=True, type=int)
+    parser.add_argument("--range-end", required=True, type=int)
+    parser.add_argument("--headers", required=True, type=str)
+    parser.add_argument("--download-path", required=True, type=str)
+    parser.add_argument("--http-uri", required=True, type=str)
+    parser.add_argument("--temp-file", required=True, type=str)
+    return parser.parse_args()
+
+
+def main():
+    file_path = os.path.join(os.path.dirname(__file__), "request_utils.py")
+    module_name = "mlflow.utils.request_utils"
+
+    spec = importlib.util.spec_from_file_location(module_name, file_path)
+    module = importlib.util.module_from_spec(spec)
+    sys.modules[module_name] = module
+    spec.loader.exec_module(module)
+    download_chunk = getattr(module, "download_chunk")
```

Comment:
```suggestion
    download_chunk = module.download_chunk
```

We can use `getattr` for an optionally-defined attribute. `download_chunk` should always exist.